Userland/Utilities/lzcat.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023, Tim Schumacher <timschumi@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibArchive/Lzma.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    StringView filename;
+
+    Core::ArgsParser args_parser;
+    args_parser.set_general_help("Decompress and print an LZMA archive");
+    args_parser.add_positional_argument(filename, "File to decompress", "file");
+    args_parser.parse(arguments);
+
+    NonnullOwnPtr<Stream> stream = TRY(Core::File::open(filename, Core::File::OpenMode::Read));
```

Comment:
Thoughts on using that "open file or standard stream" thing here? or does Posix/Linux lzcat not support `-` as an arg.